marlowe-runtime/tx-api/Language/Marlowe/Runtime/Transaction/Api.hs

```diff
@@ -61,36 +77,186 @@ import Language.Marlowe.Runtime.ChainSync.Api
   , Assets
   , BlockHeader
   , Lovelace
-  , Metadata
+  , Metadata(..)
   , PlutusScript
-  , PolicyId
+  , PolicyId(..)
   , ScriptHash
   , SlotNo
   , StakeCredential
-  , TokenName
+  , TokenName(..)
   , TxId
   , TxOutRef
   , getUTCTime
+  , parseMetadataBytes
+  , parseMetadataList
+  , parseMetadataMap
+  , parseMetadataText
   , putUTCTime
   )
 import Language.Marlowe.Runtime.Core.Api
 import Language.Marlowe.Runtime.History.Api (ExtractCreationError, ExtractMarloweTransactionError)
 import Network.Protocol.Handshake.Types (HasSignature(..))
 import Network.Protocol.Job.Types
+import qualified Network.URI
 
--- CIP-25 metadata
-newtype NFTMetadata = NFTMetadata { unNFTMetadata :: Metadata }
+data NFTMetadataFileDetails = NFTMetadataFileDetails
```

Comment:
Can we drop the trailing `Details` from the name? `Details` / `Info` / `Data` are usually redundant name suffixes.
```suggestion
data NFTMetadataFile = NFTMetadataFile
```

Reply:
Absolutely, I'll fix it.